docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+# First Steps with DataFrames.jl
+
+The first and important part to work with DataFrames.jl, is to install the `DataFrames`
+package which is available thorugh the Julia package system and can be installed using the
+following commands:
+
+```julia
+julia> using Pkg
+julia> Pkg.add("DataFrames")
+```
+
+Or,
+
+```julia
+julia> ] # `]` should be pressed 
+
+(@v1.6) pkg>  add DataFrames
+```
+
+If you want to make sure everything works as expected you could run the tests bundled
+with DataFrames.jl, but be warned that it will take more than 30 minutes.
+```julia
+julia> using DataFrames
+julia> using Pkg
+julia> Pkg.test("DataFrames")
+```
+
+Additionally, it is recommended to check the version installed with `st` command(this tutorial is for 1.x releases).
+```julia
+julia> ]
+
+(@v1.6) pkg> st 
```

Comment:
```suggestion
(@v1.6) pkg> st DataFrames
```
+ I assume that this command prints some output - right?

Reply:
yes

Reply:
yes - this is the point of this example to show which version of DataFrames.jl is used in the tutorial so that the user can cross check if all is OK on user's side